### DISCUSSION TITLE
Searching special character is not working

### DISCUSSION THREAD

**Discussion by chkp-baselz** [oracle/opengrok#4641] [Q&A]
On Opengrok Help page if I want to search for a special character such as double quotes for example (") I need to search \".
So to search "pxe" (including double quotes) I need to search ![image](https://github.com/oracle/opengrok/assets/77238449/f45353ae-597e-4fd7-8591-da216c405515)
The issue is that it shows results for pxe (without double quotes)
![image](https://github.com/oracle/opengrok/assets/77238449/6a73e073-6626-4e9a-850f-39c42989c36c)

I'm using opengrok-1.13.7

**Comment by chkp-baselz** [comment]
any update?

**Reply by vladak** [reply]
I think in general the indexer strips double quotes from the indexed terms.
